spring-ai-docs/src/main/antora/modules/ROOT/pages/api/vectordbs/chroma.adoc

```diff
@@ -98,7 +98,7 @@ List <Document> documents = List.of(
     new Document("You walk forward facing the past and you turn back toward the future.", Map.of("meta2", "meta2")));
 
 // Add the documents
-vectorStore.add(List.of(document));
+vectorStore.add(documents);
```

Comment:
`List.of(document)` is probably clearer. It is up to the user if they want to extract that to a variable. What do you think?